modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -685,68 +685,6 @@ m|WARNING
 m|DEPRECATION
 |===
 
-.Repeated variable length relationship variable inside the same pattern, with same relationship type (when run on a 5.x release)
```

Comment:
I wonder if we should remove these or if we just should change it to `(when run on version between 5.0-5.5)`. What do you think @renetapopova @hedengran ?

Reply:
I would suggest we get rid of them completely anyways, since we are un-deprecating. If we leave them in I'm sure people will interpret that as being deprecated in `5.0-5.5` and then completely removed in `>5.5`.

Reply:
I see your point and I think I agree. However, it's not great if they get the notification and can't find it in our documentation (when we say that we list all notifications). But I still think I agree.

Reply:
But wouldn't they be looking at the 5.5 version of the doc then?